tools/languagelocalizer/languageLocalizer.js

```diff
@@ -228,6 +153,12 @@ const LanguageLocalizer = {
     languageData.translations = {};
 
     if (!languageData.wikiDataID) {
+      if(this.options.check){
+        console.error(`${ languageData.languageCode } - No Wikidata ID found.`)
```

Comment:
Let's only print this and exit out if `(!languageData.wikiDataID && !languageData.alertnativeDataWikiID)`. As is, this will fail (I think) on languages like `es_CL` and `fy_NL` which are both `wikiDataID: null` but have valid values for `alternativeDataWikiID` and are not new languages.

Reply:
@lesleyjanenorton - Do you know what the point is of the alternativeDataWikiID? 

```
<http://www.wikidata.org/entity/${
        languageData.wikiDataID ||
        languageData.alternativeWikiDataID}>
``` the sparql query seems to or them either way. 

Looking at that, i think we should enforce 
```ts
{
 wikidataid:string,
 alternativeDataWikiID: string?
}

´´´